packages/api/src/settings/validators.ts

```diff
@@ -13,6 +13,22 @@ export function fieldTypeValidator(
     return isNumeric(this.value);
   }
 
+  if (this.type === 'Textarea' && this.key === 'PRAISE_FAQ') {
```

Comment:
Yes, we want to validate the input. But, is this the place? Perhaps it would be ok to do a general "Valid JSON" check and introduce JSON as a new settings type? What do you say @mattyg?

Reply:
Hmm. I agree there should be a new setting type. But let's make it specifically for a JSON file that formatted as a "question-answer" list, say  "QuestionAnswerJSON".

I think it makes sense to validate both that it's valid JSON, and that the valid JSON has the expected data fields. We could use json-schema to define the expected schema and validate it.

I agree it feels a bit weird to validate the "question-answer" format here, but I'm not sure if there is a better place, since the data is input as a json text via the settings page.

Reply:
Honestly it seems kinda weird in general to have this data be input as a json file. Do we expect end-users will have an easy time editing json documents? They're kinda particular and require very specific formatting (like no trailing commas, etc.). Or do we expect to include a default FAQ page that will likely not be edited (or only very rarely) by end-users?

If we expect they'll want to do a lot of customization of the FAQ, would it make more sense to have a UI widget for manually adding each individual question and answer? Then setting up another set of api entities, controllers, and routes for adding, removing, and editing questions and answers? We could include markdown formatting for answers too. 

Perhaps that should be stage 2 of this feature, to keep the scope contained.

Reply:
I think markdown would be better.

Reply:
We could outsource this, and just ask users to add a link to a HackMD or a github gist or a markdown file

Reply:
What we decided about this? Mitch said in the meeting that he is fine with json file format..
So should i create a new setting type and validate that input is valid JSON and also that it is in question-answer format?

Reply:
> What we decided about this? Mitch said in the meeting that he is fine with json file format.. So should i create a new setting type and validate that input is valid JSON and also that it is in question-answer format?

Yes, I think that's fine for v1 of this feature. Down the road we can consider implementing markup and a better UX for changing the contents.

For this PR yeah lets just do exactly what you said: create a new setting type which will validate that the input is JSON and in the question-answer format.